### PR TITLE
Update "remove a term from the obsoletion list"

### PR DESCRIPTION
**Pull request by sabrinatoro**
related to #6547

### CODE REVIEW

**Comment by sabrinatoro** [comment]
@nicolevasilevsky @monicacecilia 
Please review. I will add this to the curation call for discussion.

**Comment by nicolevasilevsky** [comment]
we discussed this on the call - it's fine to merge this @sabrinatoro as long as you are willing to enforce this and communicate it to the community.

**Comment by sabrinatoro** [comment]
I am VERY uncomfortable with saying that an SOP is in place as long as one person is willing to enforce it. 

We either decide that this proposal is the SOP to remove terms from the obsoletion list "because it is the right thing to do for mondo", or we decide that the SOP is to save terms from obsoletion every time someone asks (without any additional discussions). 
Maybe we need to consider how much additional work this SOP will add? We need to discuss this again before we merge and communicate with users. 
@nicolevasilevsky @monicacecilia @cmungall @matentzn 

Note: I don't mind doing the work. I just don't want SOP relying on me (or anyone) not getting hit by a bus or winning the lottery and disappearing in nature (because life is just crazy and full of surprises)

File: docs/editors-guide/curation-workflow-overview.md
Comment (matentzn) [inline docs/editors-guide/curation-workflow-overview.md:50]: I dont understand this bullet point here? justification is: because it groups children?

File: docs/editors-guide/curation-workflow-overview.md
Comment (matentzn) [inline docs/editors-guide/curation-workflow-overview.md:54]: 1 year is a long time! Generous, but I don't mind.

**Comment by ManuelFranco1113** [comment]
I just want to say Thank You to everyone who supported me through the years. My name is Manuel Franco, New Berlin, Wisconsin. My story of how I won the Powerball lottery of $768.4M is a bit of a tale. I have been playing Powerball tickets for 6 years now since I turned 18. I bought my first ticket on my 18 birthday. I was feeling very lucky that day because I had contacted Dr. Odunga Michael to help me with the winning Powerball numbers. I really had that great great feeling that I looked at the camera wanting to wink at it. I only did a tiny part of it and trusted him. He gave me the numbers after I played a couple other tickets along with it for $10. I checked my ticket after the winnings came online and saw the numbers were correct including the Power play. I screamed for about 10 minutes because it felt like a dream. I had won $768.4M. You can check my winning testimony with the lottery officials just with my name search. Thank you Dr Odunga. Well, his email is [odungaspelltemple@gmail.com](mailto:odungaspelltemple@gmail.com) and you can also call or Whats-app him at +2348167159012 so you guys can contact him

**Comment by sabrinatoro** [comment]
@nicolevasilevsky @twhetzel @monicacecilia

File: docs/editors-guide/curation-workflow-overview.md
Comment (nicolevasilevsky) [reply docs/editors-guide/curation-workflow-overview.md:50]: @matentzn I think what is meant here is we prefer not to create grouping classes where we'd have to manually classify the children (Chris has brought this up many times). If the grouping class had a logical definition and terms were autoclassified, that would be ideal.

This seems like something the curation team should be aware of.

Comment (matentzn) [reply docs/editors-guide/curation-workflow-overview.md:50]: It seems a bit weak of a reason to me to say: "because we would have to manually classify, we don't want to", or "because we cant have a logical definition, we don't want to keep the grouping class". I think we should make a stronger point here.

File: docs/editors-guide/curation-workflow-overview.md
Comment (twhetzel) [reply docs/editors-guide/curation-workflow-overview.md:54]: yes, one year does seem like a long time

File: docs/editors-guide/curation-workflow-overview.md
Comment (twhetzel) [inline docs/editors-guide/curation-workflow-overview.md:52]: > removed from the obsoletion list

perhaps also add "or no longer tagged as an obsoletion candidate"

> further discussion with the Mondo team and the user community

where will this further discussion happen and how will the community be notified?

File: docs/editors-guide/curation-workflow-overview.md
Comment (twhetzel) [reply docs/editors-guide/curation-workflow-overview.md:54]: > requests for delaying term obsoletion

how is this request made?

> obsoletion candidates will eventually be obsoleted from Mondo.

this point starts with talking about waiting to obsolete a term if a user/community requests to pause the obsoletion, but then ends with the term will eventually be obsoleted so is this statement intended to say that a request can be made to pause an obsoletion but that extension is only for up to one year or presence of active discussion (might be good to state where this active discussion or placeholder of this discussion like a tracker item should be)?

**Comment by sabrinatoro** [comment]
The document was updated based on the discussion in the comments and in meetings. 
@twhetzel and/or @nicolevasilevsky please review and if approved, merge this PR. Thank you. 
Additional discussions about SOP and practicalities will need to be discussed on a call.

**Review by nicolevasilevsky — APPROVED** [review]
this looks fine to me

**Pull request merged**
